### ISSUE TITLE
Unable to save to PNG: Uncaught TypeError: The "path" argument must be one of type string, Buffer, or URL. Received type object

### ISSUE THREAD

**Issue by qchenevier** [sverweij/atom-graphviz-preview-plus#38]
When I try to save the preview in PNG, I get this error message:

<img width="459" alt="Screenshot 2020-10-22 at 16 43 56" src="https://user-images.githubusercontent.com/24975491/96888364-d6b00d80-1485-11eb-9051-a73dc84ff720.png">

Full message with the stacktrace:
```
Uncaught TypeError: The "path" argument must be one of type string, Buffer, or URL. Received type object

internal/fs/utils.js:426

TypeError [ERR_INVALID_ARG_TYPE] [ERR_INVALID_ARG_TYPE]: The "path" argument must be one of type string, Buffer, or URL. Received type object
    at Object.openSync (fs.js:439:10)
    at Object.func (electron/js2c/asar.js:138:31)
    at Object.func [as openSync] (electron/js2c/asar.js:138:31)
    at Object.writeFileSync (fs.js:1206:35)
    at Proxy.writeFileSync (/Users/XXXX/.atom/packages/graphviz-preview-plus/node_modules/fs-plus/lib/fs-plus.js:292:17)
    at /Users/XXXX/.atom/packages/graphviz-preview-plus/lib/graphviz-preview-plus-view.js:494:28
    at HTMLImageElement.<anonymous> (/Users/XXXX/.atom/packages/graphviz-preview-plus/lib/svgToRaster.js:47:13)
```

System:
```
OS: macOS Catalina 10.15.6
Atom: 1.52.0
graphviz-preview-plus: 1.5.11
```

**Comment by sverweij** [comment]
@qchenevier confirmed - thanks for raising this. I'm looking into a solution.

**Comment by sverweij** [comment]
@qchenevier found the root cause + a fix. Hope to have a new version up by the end of the weekend.

**Comment by qchenevier** [comment]
That's impressive, man.

**Comment by sverweij** [comment]
@qchenevier it should work again as intended - I've published 1.15.12 that includes the fix.

**Comment by qchenevier** [comment]
This is working well. Thanks & bravo
